graph/pull.go

```diff
@@ -538,14 +538,14 @@ func (s *TagStore) pullV2Tag(r *registry.Session, out io.Writer, endpoint *regis
 	out.Write(sf.FormatStatus(tag, "Pulling from %s", repoInfo.CanonicalName))
 
 	// downloadInfo is used to pass information from download to extractor
+	// tmpFile will be set if and only if the download was successful
```

Comment:
previously, we checked both `downloaded` and `tmpFile`, but not quite correctly, so I just simplified by getting rid of `downloaded` since it provides no extra information, just opportunity for mistakes.